tests/valgrind-check/run_checks.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+
+function check_with_valgrind() {
+  bash tests/valgrind-check/valgrind.sh
+}
+
+cd "$(dirname $0)"/../../
+
+failure=0
+check_with_valgrind || { echo "FAIL: valgrind check failed"; whoami; failure=1; }
```

Comment:
I'd prefer `echo "FAIL: valgrind check failed [$(whoami)]"` or something like that.

Reply:
This is just a leftover from debugging, removed.